spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -87,30 +87,67 @@
 
   it "does not fail when the schema matches" do
     create_schema("array_schema", {
-      "type" => "array",
-      "items" => { "type" => "string" },
+      "type": "array",
+      "items": { "type": "string" },
     })
 
     expect(response_for(["valid"])).to match_response_schema("array_schema")
   end
 
   it "supports $ref" do
-    create_schema("single", {
-      "type" => "object",
-      "required" => ["foo"],
-      "properties" => {
-        "foo" => { "type" => "string" },
+    create_schema("user", {
+      "id": "file:/#{JsonMatchers.schema_root}/user.json#",
+      "type": "object",
+      "required": ["id"],
+      "properties": {
+        "id": {
+          "type": "integer"
+        }
       }
     })
-    create_schema("collection", {
-      "type" => "array",
-      "items" => { "$ref" => "single.json" },
+    create_schema("users", {
+      "id": "file:/#{JsonMatchers.schema_root}/users.json#",
```

Comment:
I wasn't able to get these tests to pass when I used relative paths for the `id`s. Any ideas?